lib/workload/stateless/statelessStackCollectionClass.ts

```diff
@@ -14,12 +14,14 @@ import {
   SequenceRunManagerStack,
   SequenceRunManagerStackProps,
 } from './stacks/sequence-run-manager/deploy/component';
+import { BsRunsUploadManagerStackProps } from './bs_runs_upload_manager/deploy/lib/stacks/bs_runs_upload_manager_stack';
 
 export interface StatelessStackCollectionProps {
   postgresManagerStackProps: PostgresManagerStackProps;
   metadataManagerStackProps: MetadataManagerStackProps;
   sequenceRunManagerStackProps: SequenceRunManagerStackProps;
   fileManagerStackProps: FilemanagerProps;
+  bsRunsUploadManagerStackProps: BsRunsUploadManagerStackProps;
 }
 
 export class StatelessStackCollection {
```

Comment:
@alexiswl you will need to add the stack as the class property of this class, and initiate it at the constructor to be part of the pipeline stack.

```ts

// on line 33
readonly bsRunsUploadManagerStack: Stack

// and on line 57
this.bsRunsUploadManagerStack = new BsRunsUploadManagerStack(....)

```

Reply:
Done in b4041ad7905c37aaccd6af7fb0f9180cb1910213